rest-api/common-rest/src/main/java/uk/gov/gchq/gaffer/rest/service/v2/AbstractOperationService.java

```diff
@@ -54,6 +58,11 @@ public Set<Class <? extends Operation>> getSupportedOperations() {
         return getGraphFactory().getGraph().getSupportedOperations();
     }
 
+    public Set<String> getAllOperationsNames() {
```

Comment:
This should be the same type as `getSupportedOperations` (`Set<Class <? extends Operation>>`) and should therefore be called `getAllOperations`.

It would be better if instead this was merged into `getSupportedOperations` with a `final boolean includeUnsupported` used to decide if unsupported are included. As done for the Operation details.